doc/python/lines-on-maps.md

```diff
@@ -111,6 +111,33 @@ fig.update_layout(
 
 fig.show()
 ```
+### High performance US Flight Paths Map
+If you can relinquish having individual styles for the flight paths (e.g. opacity), you can put multiple paths into one trace, which makes the map render much faster. 
```

Comment:
About the wording I'm hesitating between `faster` and `much faster` here... Do you have orders of magnitude for the two solutions?